python/ray/data/_internal/execution/operators/base_physical_operator.py

```diff
@@ -108,7 +118,7 @@ def get_transformation_fn(self) -> AllToAllTransformFn:
         return self._bulk_fn
 
     def progress_str(self) -> str:
-        return f"{len(self._output_buffer)} output"
+        return f"{self._output_rows} rows output"
```

Comment:
Should this just call `num_output_rows_total`?

Reply:
Good suggestion! That is better, I add or 0 here as when `self._output_rows` is 0, `self.input_dependencies[0].num_output_rows_total()` will be None, 
so in this case without `or 0`, it will be 
![image](https://github.com/user-attachments/assets/48eab96e-0c22-4892-bdc3-12f3aa040086)
in the begining.